### DISCUSSION TITLE
Add support for Django

### DISCUSSION THREAD

**Discussion by alexandernst** [cloudflare/workerd#1934] [Python Packages]
Title says it all.

**Comment by openfordream** [comment]
DRF AS WELL.

**Reply by knowsuchagency** [reply]
DRF is fine but the real game-changer would be https://django-ninja.dev/ IMHO

**Comment by ryanking13** [comment]
Django is now supported in Python workers: https://developers.cloudflare.com/workers/languages/python/packages/django/
